packages/ai-persistence-cloudflare/tests/d1.test.ts

```
@@ -0,0 +1,238 @@
+/// <reference types="@cloudflare/workers-types" />
+import { describe, expect, it } from 'vitest'
+import { DatabaseSync } from 'node:sqlite'
+import { EventType } from '@tanstack/ai'
+import type { StreamChunk } from '@tanstack/ai'
+import {
+  cloudflarePersistence,
+  createD1Driver,
+  createR2ArtifactStore,
+} from '../src/index'
```

ESLint

[error] `../src/index` import should occur before type import of `@tanstack/ai`

(import/order)